command/agent/command.go

```diff
@@ -70,6 +70,18 @@ func (c *Command) readConfig() *Config {
 	// Server-only options
 	flags.IntVar(&cmdConfig.Server.BootstrapExpect, "bootstrap-expect", 0, "")
 
+	// Client-only options
+	flags.StringVar(&cmdConfig.Client.StateDir, "state-dir", "", "")
+	flags.StringVar(&cmdConfig.Client.AllocDir, "alloc-dir", "", "")
+	flags.StringVar(&cmdConfig.Client.NodeID, "node-id", "", "")
+	flags.StringVar(&cmdConfig.Client.NodeClass, "node-class", "", "")
+
+	var servers string
+	flags.StringVar(&servers, "servers", "", "")
```

Comment:
Hmm... I see why you did this, as we already have a `-server` arg. It would be nice if we could think of a reasonable name for a flag that you could specify multiple times, but for my lack of creativity this should suffice.